packages/hawtio/src/plugins/connect/remote/Remote.tsx

```diff
@@ -218,19 +227,23 @@ const ConnectionItem: React.FunctionComponent<{
           </Button>
           <Dropdown
             key={`connection-action-dropdown-${id}`}
-            isPlain
```

Comment:
sometimes it's left, sometimes it's removed on Dropdown. is there any difference?

Reply:
yeah, when is plain it's missing borders in the menu. I removed that from all dropdowns now because it looked better in my opinion.

Reply:
Can you show us the screenshots?

Reply:
Maybe changing the styles can be done under a separate issue from this refactoring to PF 5, as it's doing two different things at once if it's not just mandatory for PF 5.

Reply:
It's actually not about changing the style but rather preserving the current one. It seems that in PF5 `isPlain` changed the behaviour and it draws the whole dropdown without the border, not only the 'header': dropdown with keeping `isPlain`(on the left PF5 with isPlain): 
<img width="603" alt="Screenshot 2024-06-06 at 15 43 42" src="https://github.com/hawtio/hawtio-next/assets/6814482/45435535-617e-4803-828a-5f955e8f6f50">

dropdown without `isPlain`:
<img width="1135" alt="Screenshot 2024-06-06 at 15 40 57" src="https://github.com/hawtio/hawtio-next/assets/6814482/b6b807ff-36e0-4618-80c1-35612e818ffa">
so as you can see the last picture removing `isPlain` is completely fine as is preserves the current look and feel and that's why I removed them.

Only change is in the header where there is a thin side border:
<img width="472" alt="Screenshot 2024-06-06 at 15 50 14" src="https://github.com/hawtio/hawtio-next/assets/6814482/745989c8-a14a-430b-a425-533baaaff766">

But it follows the PF5 quickstarts.

Reply:
Makes sense. Let's remove all the `isPlain` here then.